### DISCUSSION TITLE
Add Expo router starter on tamagui CLI

### DISCUSSION THREAD

**Discussion by bidah** [tamagui/tamagui#1836] [Ideas]
Hello 👋,

The Tamagui CLI offers options for starting with Next.js, Expo, Solito, Simple Web (Vite), and Takeout. I think it would be a great improvement to add an Expo option that includes TypeScript and Expo Router.

Expo Router with Metro Web is a very powerful combination, and it works perfectly on native and web.

There was an old discussion from a few months ago (https://github.com/tamagui/tamagui/discussions/616) where people couldn't make this stack work, but not anymore. This is probably a combination of Metro Web getting better and the Tamagui team making the fixes.

I would be happy to help with this integration. Let me know what you think.

**Comment by medv** [comment]
You can use this as a reference https://github.com/michaelessiet/expo-tamagui-template
Does not answer your actual request, but maybe it helps anyway
